src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.skia.cs

```diff
@@ -141,8 +130,8 @@ SkiaGeometrySource2D BuildGeometry()
 				}
 				else if (background is ImageBrush imgBackground)
 				{
-					//var uiImage = imgBackground.ImageSource?.ImageData;
-					//if (uiImage != null && uiImage.Size != CGSize.Empty)
+					//var imgSrc = imgBackground.ImageSource;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932478)